src/intellisense/serviceRegistry.node.ts

```diff
@@ -16,4 +21,15 @@ export function registerTypes(serviceManager: IServiceManager, _isDevMode: boole
         IExtensionSingleActivationService,
         PythonKernelCompletionProviderRegistration
     );
+    serviceManager.addSingleton<IExtensionSyncActivationService>(
```

Comment:
Moved a bunch of stuff here from the 'notebooks' serviceRegistry. Seemed weird that notebooks would register stuff from the intellisense folder.